tests/unit/OptionsListUtilsTest.ts

```diff
@@ -865,61 +857,50 @@ describe('OptionsListUtils', () => {
             }, []);
 
             const options = OptionsListUtils.getShareDestinationOptions(filteredReportsWithWorkspaceRooms, OPTIONS.personalDetails, []);
-            const filteredOptions = OptionsListUtils.filterOptions(options, 'Mutants Lair');
+            const filteredOptions = OptionsListUtils.filterAndOrderOptions(options, 'Mutants Lair');
 
             expect(filteredOptions.recentReports.length).toBe(0);
         });
 
         it('should show the option from personal details when searching for personal detail with no existing report (getOptions)', () => {
-            const options = OptionsListUtils.getOptions({reports: OPTIONS.reports, personalDetails: OPTIONS.personalDetails});
-            const filteredOptions = OptionsListUtils.filterOptions(options, 'hulk');
+            const options = OptionsListUtils.getValidOptions({reports: OPTIONS.reports, personalDetails: OPTIONS.personalDetails});
+            const filteredOptions = OptionsListUtils.filterAndOrderOptions(options, 'hulk');
 
             expect(filteredOptions.recentReports.length).toBe(0);
 
             expect(filteredOptions.personalDetails.length).toBe(1);
             expect(filteredOptions.personalDetails.at(0)?.login).toBe('brucebanner@expensify.com');
         });
 
-        it('should return all matching reports and personal details (getOptions)', () => {
-            const options = OptionsListUtils.getOptions({reports: OPTIONS.reports, personalDetails: OPTIONS.personalDetails}, {maxRecentReportsToShow: 5});
-            const filteredOptions = OptionsListUtils.filterOptions(options, '.com');
-
-            expect(filteredOptions.recentReports.at(0)?.text).toBe('Captain America');
-
-            // We expect that only personal details that are not in the reports are included here
-            expect(filteredOptions.personalDetails.length).toBe(4);
-            expect(filteredOptions.personalDetails.at(0)?.login).toBe('natasharomanoff@expensify.com');
-        });
```

Comment:
Why was this removed? Is it redundant? If so where is it covered now?

Reply:
Yes, this test was consolidated into this test here:

https://github.com/margelo/expensify-app-fork/blob/2eaeaf42ed7696ff3dc98b34567ee17be4cd2f5c/tests/unit/OptionsListUtilsTest.ts#L947-L957